package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "material-ui": "^0.20.0",
+    "material-ui-search-bar": "^0.4.1",
```

Comment:
think you won't need that after all. we made a search bar without this haha

Reply:
haha okey dokey... I will have a look :smile:

Reply:
:+1:  I removed this ... will push